packages/docusaurus-theme-classic/src/theme/hooks/useLogo.js

```diff
@@ -15,7 +15,7 @@ const useLogo = () => {
     siteConfig: {baseUrl, themeConfig: {navbar: {logo = {}} = {}}} = {},
   } = useDocusaurusContext();
   const {isDarkTheme} = useThemeContext();
-  const logoLink = logo.href || baseUrl;
+  const logoLink = useBaseUrl(logo.href || baseUrl);
```

Comment:
Wouldn't the `useBaseUrl(baseUrl)` case result in duplication in the case of non-`/` `baseUrl`s?

Reply:
Should we just be wrapping around `logo.href`?

Reply:
Oops, fixed.
I set `/` as the default value which will be processed by `useBaseUrl`


https://github.com/facebook/docusaurus/blob/be7367b2f85c366c6343787a78f80ab1c75397b0/packages/docusaurus/src/client/exports/useBaseUrl.ts#L30-L32